12-genetic-chromosome-collision/README.md

```diff
@@ -1,7 +1,8 @@
 # Using Genetic Algorithm to model Chromosome Collision
 
 ## Team Members:
-Names (Student Numbers)
+ (94122681004) محمدعلی الاشقر
+
 
 ## Descreption:
-To be Defined
+پروژه من مسیر یابی وساخت کروموزوم - الگوریتم ژنیتیک
```

Comment:
1) Please provide a description for one paragraph (preferably in English) to introduce how your application works.
2) If any other libraries or files needed to run the code, mention them in this page